Library/Formula/rtmidi.rb

```diff
@@ -1,23 +1,17 @@
 class Rtmidi < Formula
   desc "C++ classes that provide a common API for realtime MIDI input/output"
   homepage "https://www.music.mcgill.ca/~gary/rtmidi/"
-  url "https://github.com/thestk/rtmidi/archive/2.1.0.tar.gz"
-  sha256 "52e6822fc413d5d3963c5b7bfe412ed69233bb81a7f04d6097f5b56aafa28934"
-
-  bottle do
```

Comment:
Don't worry too much about it here but you don't need to wipe the `bottle do` block on updates.